src/main/java/codeu/model/data/User.java

```diff
@@ -35,13 +36,15 @@
      * @param passwordHash the password hash of this User
      * @param creation     the creation time of this User
      * @param bio          the biography of this User
+     * @param language     the language of this User
      */
-    public User(UUID id, String name, String passwordHash, Instant creation, String bio) {
+    public User(UUID id, String name, String passwordHash, Instant creation, String bio, String language) {
```

Comment:
Just a couple things to think about, no changes required: what if a user speaks multiple languages? What if they learn a new language?

Reply:
Another thing- what happens if their formatting for the languages is wrong? is it case sensitive?

Reply:
Also -- just an overall question for the language specification: is this intended to translate the whole site or just messages?